### ISSUE TITLE
feat: support for text templates

### ISSUE THREAD

**Issue by suterma** [slipmatio/control-knob#17]
Currently the displayed value is just a simple text. It's conceivable that a user wants to express a percent value or a sound pressure level (dB) or something else. A text template could support this.

@Uninen: I have already a solution for this in my fork's main repo. You might take it from there or, if you prefer, I can create a PR in a separate feature branch.

**Comment by Uninen** [comment]
Yes, I agree, the display value should be fully configurable. A function or a template would be one possible solution but I'd also like to have a fully headless implementation of the whole component so that you could use slots and arbitrary markup to make it look whatever you like.

**Comment by suterma** [comment]
Sounds interesting. I didn't know that slots in SVG's are possible.
